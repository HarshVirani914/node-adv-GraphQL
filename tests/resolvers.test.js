const resolvers = require("../src/resolvers");

describe("resolvers", () => {
  test("feed", async () => {
    const result = await resolvers.Query.feed(null, null, {
      models: {
        Post: {
          findMany: jest.fn(() => [{ message: "hello" }]),
        },
      },
    });
    expect(result).toMatchSnapshot();
  });

  test("posts", async () => {
    const result = await resolvers.Query.posts(null, null, {
      user: { id: 1 },
      models: {
        Post: {
          findMany: jest.fn(() => [{ message: "hello" }]),
        },
      },
    });
    expect(result).toMatchSnapshot();
  });
});
