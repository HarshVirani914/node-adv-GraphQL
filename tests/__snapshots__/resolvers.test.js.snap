// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resolvers feed 1`] = `
Array [
  Object {
    "message": "hello",
  },
]
`;

exports[`resolvers posts 1`] = `
Array [
  Object {
    "message": "hello",
  },
]
`;
